src/maks.kulykov/DatabaseManager.java

```diff
@@ -0,0 +1,48 @@
+package maks.kulykov;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+
+public class DatabaseManager {
+    private String db = "mydb";
+    private String userName = "postgres";
+    private String password = "postgres";
+
+    private String cmdDB;
```

Comment:
I think you should be consistent in the naming style - camelCase. So I think it is better 'cmdDb'.

Reply:
+